report/renderer/report-ui-features.js

```diff
@@ -318,20 +318,24 @@ export class ReportUIFeatures {
    * @return {Array<HTMLElement>}
    */
   _getThirdPartyRows(rowEls, finalDisplayedUrl) {
-    /** @type {Array<HTMLElement>} */
-    const thirdPartyRows = [];
     const finalDisplayedUrlRootDomain = Util.getRootDomain(finalDisplayedUrl);
+    const mainEntityName = this.json.entityClassification?.firstParty;
 
+    /** @type {Array<HTMLElement>} */
+    const thirdPartyRows = [];
```

Comment:
drive by suggestion since you're touching this code: `thirdPartyRowEls`, to match how we like to name element variables?